koku/masu/processor/orchestrator.py

```diff
@@ -373,7 +374,20 @@ def start_manifest_processing(  # noqa: C901
                     queue=SUBS_EXTRACTION_QUEUE
                 )
                 LOG.info(log_json("start_manifest_processing", msg="created subs_task signature", schema=schema_name))
-                async_id = chord(report_tasks, group(summary_task, hcs_task, subs_task))()
+                ocp_on_cloud_trino_task = process_openshift_on_cloud_trino.s(
```

Comment:
Do we want to utilize an unleash flag here as a way to soft roll this out for select customers first or to provide a way to disable the logic in case we experience delays?